### ISSUE TITLE
clean up cell_def_cycle_tab.py

### ISSUE THREAD

**Issue by rheiland** [PhysiCell-Tools/PhysiCell-Studio#277]
Now that CycleTab is a subclass of CellDefSubTab, let's eventually clean up confusing uses of objects, e.g., `self.celldef_tab.param_d` vs. `self.param_d`

**Comment by drbergman** [comment]
Yeah, I feel like I just shoulda done all this or kept it all as one file. Instead, I left it in this weird state...
